Dockerfile-drupal

```diff
@@ -55,8 +55,7 @@ RUN apt-get update \
 RUN update-alternatives --set php /usr/bin/php8.1
 
 RUN pecl channel-update pecl.php.net \
-    && pecl download imagick \
-    && pecl install imagick memcached
+    && pecl install memcached
```

Comment:
We should probably try removing memcached one of these days too, as it's never been used.